### PR TITLE
fix: Constants and test code resolved

### PR DESCRIPTION
**Pull request by AxithChoudhary**

### CODE REVIEW

File: featched_data.txt [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  'seller_information': [{'Myntra': ([{'text': 'Flat ₹200 off above 999 on First OrderUse Code: MYNTRA200'},
+      {'text': 'Delivery by Thu, 4 Jul'},
+      {'text': 'Spend ₹1,199 for free delivery'}],
+     {'base_price': '₹1,049.00'},
+     {'total_price': '₹1,128.00'})},
+   {'Flipkart': ([{'text': 'Free delivery by Mon, 8 Jul'}],
+     {'base_price': '₹1,000.00'},
+     {'total_price': '₹1,000.00'})},
+   {'Nykaa Man': ([{'text': 'Free delivery by Thu, 11 Jul'}],
+     {'base_price': '₹1,200.00'},
```
Comment (ShubhamVora-1206) [inline featched_data.txt:171]: Avoid pushing dummy data, keep it on local but don't push it with other changes

**Review by ShubhamVora-1206 — APPROVED** [review]
Prompt can be refined, meanwhile this branch can be merged to continue with the development.

File: featched_data.txt [hunk 2]
```diff
@@ -0,0 +1,172 @@
+[{'position': 1,
+  'title': "Levi'S Solid Men Polo Neck Red T-shirt",
```
Comment (gaurav-9721) [inline featched_data.txt:2]: Gitignore these files please

File: server/services/utils/prompts/preprocessing_prompt.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    [0, 2]
+
+    EXAMPLE
+    user_query : mens black formal shoes size 9
+    product_list : ['mens black formal shoes size 9', 'mens black formal shoes size 8', 'mens black casual shoes size 9', 'mens shoes', 'women black formal shoes size 9']
+    
+    [0]
+
+    do not return anything else, only the index. do not return ```python ```
```
Comment (gaurav-9721) [inline server/services/utils/prompts/preprocessing_prompt.py:33]: Use capital letter after full stop.

**Review by gaurav-9721 — APPROVED** [review]

**Comment by gaurav-9721** [comment]
Merged AC/pricing changes to GV/structure_project

**Review by gaurav-9721 — APPROVED** [review]

**Pull request merged**